features/networking/multus.feature

```diff
@@ -240,27 +223,185 @@ Feature: Multus-CNI related scenarios
     And the output should contain "Hello OpenShift"
 
     # Check that the link is removed from node
-    Given I run the :exec admin command with:
-      | pod              | <%= cb.target_sdnpod %> |
-      | n                | openshift-sdn           |
-      | oc_opts_end      |                         |
-      | exec_command     | sh                      |
-      | exec_command_arg | -c                      |
-      | exec_command_arg | ip link show            |
+    When I run network command on the "<%= cb.target_node %>" node's sdn pod:
+      | sh | -c | ip link show |
     Then the step should succeed
-    And the output should not contain "eth08"
+    And the output should not contain "<%= cb.nic_name %>"
 
     # Delete the pod and check the link on the node again
     When I run the :delete client command with:
       | object_type | pod                  |
       | l           | name=host-device-pod |
     Then the step should succeed
-    Given I run the :exec admin command with:
-      | pod              | <%= cb.target_sdnpod %> |
-      | n                | openshift-sdn           |
-      | oc_opts_end      |                         |
-      | exec_command     | sh                      |
-      | exec_command_arg | -c                      |
-      | exec_command_arg | ip link show            |
+    When I run network command on the "<%= cb.target_node %>" node's sdn pod:
+      | sh | -c | ip link show |
+    Then the step should succeed
+    And the output should contain "<%= cb.nic_name %>"
+
+
+  # @author bmeng@redhat.com
+  # @case_id OCP-21854
+  @admin
+  Scenario: Create pods with muliple cni plugins via multus-cni - macvlan + macvlan
+    # Make sure that the multus is enabled
+    Given the master version >= "4.0"
+    And the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-bridge.yaml |
+      | n | <%= project.name %> |
```

Comment:
I assume this is a namespaced resource, otherwise we can again get name collisions.

Reply:
Yes, it is a namespaced resource.